core/src/main/java/bisq/core/btc/wallet/Restrictions.java

```diff
@@ -93,4 +96,10 @@ public static Coin getMinSellerSecurityDepositAsCoin() {
             SELLER_SECURITY_DEPOSIT = Coin.parseCoin("0.005"); // 0.005 BTC about 20 USD @ 4000 USD/BTC
         return SELLER_SECURITY_DEPOSIT;
     }
+
+    public static Coin getMinRefundAtMediatedDispute() {
+        if (MIN_REFUND_AT_MEDIATED_DISPUTE == null)
+            MIN_REFUND_AT_MEDIATED_DISPUTE = Coin.parseCoin("0.003"); // 0.003 BTC about 21 USD @ 7000 USD/BTC
```

Comment:
In PR #3826 we use:
`MIN_BUYER_SECURITY_DEPOSIT = Coin.parseCoin("0.006");`
`SELLER_SECURITY_DEPOSIT = Coin.parseCoin("0.006");`
So that is actually the value which `MIN_REFUND_AT_MEDIATED_DISPUTE` refers to. 
I will add a comment to make clear that the value cannot be lower than MIN_BUYER_SECURITY_DEPOSIT/SELLER_SECURITY_DEPOSIT.